### ISSUE TITLE
Use WindTerm 2.6.0 to download large files, the downloaded files will be different from the ones on the server (through md5 value comparison). However, this issue does not exist when using WindTerm 2.6.0 Prerelease 10.

### ISSUE THREAD

**Issue by lgc653** [kingToolbox/WindTerm#2230]
Use WindTerm 2.6.0 to download large files, the downloaded files will be different from the ones on the server (through md5 value comparison). However, this issue does not exist when using WindTerm 2.6.0 Prerelease 10.

**Comment by err0rcat** [comment]
I confirm this. Files downloaded via Filer are broken.

**Comment by zx900930** [comment]
same here, even small size files (less than 5MB) are currpt after download.

**Comment by duucck** [comment]
same

**Comment by QQ853695231** [comment]
so am i

**Comment by kingToolbox** [comment]
This issue has been fixed. Please download the latest version [WindTerm 2.6.0](https://github.com/kingToolbox/WindTerm/releases/tag/2.6.0) or [WindTerm 2.7.0 Prerelease 1](https://github.com/kingToolbox/WindTerm/releases/tag/2.7-prerelease). I apologize for the delay in releasing the fixed version, as I have been traveling and working without reliable internet access.

**Comment by lgc653** [comment]
Thank you, you're amazing.

**Comment by err0rcat** [comment]
@kingToolbox way to go bro! thanks! any chance for an Apple Silicon version? :)

**Comment by kingToolbox** [comment]
@err0rcat 
Yes, I have good news about this. The next version of `WindTerm` will start using `Qt 6.7`, which has great support for the ARM version of macOS. At that time, I will begin working on the ARM version. The brand new version will redesign and reimplement many features, which is a considerable amount of work. Therefore, please be patient for a little while longer.

**Comment by err0rcat** [comment]
@kingToolbox thats great news! thanks!

**Comment by jeyriku** [comment]
Hi,

Thank you for your great work on that powerful tool. Do you please have any news for the arm64 version ?
Thanks in advance !
